torch/_inductor/compile_fx.py

```diff
@@ -296,11 +298,29 @@ def compile_fx_inner(
             if isinstance(t, torch.Tensor)
         )
 
+        def _has_dynamism(t):
+            if isinstance(t, torch.Tensor):
+                if not all(isinstance(s, (int, sympy.Integer)) for s in t.size()):
```

Comment:
@ezyang said "Stop testing sympy.Integer on size lol" because he has told me this before, but I am dense.